### ISSUE TITLE
[a11y]: ai-chat should enable JAWS to announce feedback form

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#864]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS

### Description

All visual content in the Feedback form is inaccessible to screen reader users. When navigating with Arrow keys, the screen reader does not identify or announce the content, including visual headings, text, images, or icons within the modal. As a result, users cannot perceive or interact with these elements- Modal is not Screen reader readable .

Recommended fix: "Make sure all the visual content in the modal is available to screen reader users, and that all content is read out correctly using standard screen reader
methods. Headings, images, and other elements must be conveyed with their required semantics."
Note : Detailed Screen reader Testing only possible after making the content identifiable by Screen reader Technologies

User impact: Screen reader users are unable to read or understand the content and structure of the modal, severely limiting their ability to navigate and interact with the Modal.

### WCAG 2.1 Violation

WCAG Mapping: 1.3.1 Info and Relationships

### Reproduction/example

n/a

### Steps to reproduce

1. opem feddback form
2. use JAWS to announce contents of feedback form

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
